### DISCUSSION TITLE
[RFC] ConfigProvider support replace arrow icon

### DISCUSSION THREAD

**Discussion by arvinxx** [ant-design/ant-design#43959] [RFCs]
## Summary

Provide a props on ConfigProvider, pass the corresponding arrow icon ReactNode, to achieve unified replacement of arrow icons for select, collapse and other components.

在 ConfigProvider 上提供一个 props，传入相应的 arrow 图标的 ReactNode，实现统一替换 select、collapse 等组件的箭头图标。

## Motivation

The default arrow icons of select, collapse and other components are too ugly. Before antd updates the arrow icons, we need to support custom arrow icons in our business.

默认的 select、 collapse 等组件的箭头太丑了。在 antd 不自我更新箭头图标之前，让业务支持自定义箭头图标。

相关 issuse:
- https://github.com/ant-design/ant-design/issues/39529

## API

**ConfigProvider**

| Property | Description | Type | Default | 
| --- | --- | --- | --- | 
| arrowIcon | custom arrow icon | ReactNode | - | 

## Basic Example

```tsx
import { ChevronRight } from 'react-lucide';
import { ConfigProvider,Select , Dropdown } from 'antd';

export default () => (
  <ConfigProvider arrowIcon={<ChevronRight />}>
    <Select></Select>
   <Dropdown></Dropdown>
  </ConfigProvider>
)
```

**Comment by crazyair** [comment]
CP 直接增加各个组件的 props，不然需求会越来越多

**Comment by MadCcc** [comment]
https://github.com/ant-design/ant-design/issues/41413

**Comment by stefann01** [comment]
Would be nice to have such a feature to be able to change icons
